server/drivers/serialPOS_cd5220.c

```diff
@@ -0,0 +1,363 @@
+/**
```

Comment:
Why are these files new?

Reply:
Might be a bug of GitHub. Maybe a rebase or recreation of the PR is needed.

Reply:
Once you are happy with the code, I think I will just push it directly to a separate branch. Just in case this PR is actually broken somehow.